wpilib/wpilib/_impl/main.py

```diff
@@ -80,7 +80,8 @@ def run(robot_class, **kwargs):
         :param **kwargs: Keyword arguments that will be passed to the executed entry points
         :returns: This function should never return
     '''
-    
+
+    warnings.filterwarnings("default", category=DeprecationWarning, module='wpilib')
```

Comment:
I'm not sure whether we want these to unconditionally show on the roboRIO. Definitely in tests though.

Reply:
Well hopefully we never use deprecated methods in our code, and I think it'll be good to tell the users of any deprecated stuff? Although...if they *want* to use SampleRobot, they don't  want that error every time, do they? Maybe they do? Idk.

Reply:
I think yelling at them in the tests would be enough. I'm not sure I'd want my drive team to be worrying about these though.